docs/remove.md

```
@@ -0,0 +1,252 @@
+1. **Pre-Remove Hook** (if configured)
+2. **Binary Symlink Removal** from `~/.local/share/soar/bin`
+3. **Provides Symlink Cleanup** for alternative names
+4. **Desktop Entry Removal** from `~/.local/share/applications`
+5. **Icon Symlink Cleanup** from `~/.local/share/icons`
+6. **Package Directory Removal** from `~/.local/share/soar/packages`
+7. **Cache Handling**: download cache preserved (use `soar clean --cache` to reclaim)
+8. **Database Cleanup**: removes the package record and portable entries
+
+Example output:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)